src/data/dialogmotekandidat/dialogmotekandidatTypes.ts

```diff
@@ -2,3 +2,16 @@ export interface DialogmotekandidatDTO {
   readonly kandidat: boolean;
   readonly kandidatAt?: string;
 }
+
+export interface DialogmotekandidatHistorikkDTO {
+  tidspunkt: Date;
+  type: HistorikkType;
+  vurdertAv: string | null;
+}
+
+export enum HistorikkType {
+  KANDIDAT = "KANDIDAT",
+  UNNTAK = "UNNTAK",
+  IKKE_AKTUELL = "IKKE_AKTUELL",
+  LUKKET = "LUKKET",
+}
```

Comment:
Er det egentlig en vurdering og vurderingtype vi har her? 🤔
```suggestion
export interface DialogmotekandidatVurderingHistorikkDTO {
  tidspunkt: Date;
  type: HistorikkType;
  vurdertAv: string | null;
}

export enum VurderingType {
  KANDIDAT = "KANDIDAT",
  UNNTAK = "UNNTAK",
  IKKE_AKTUELL = "IKKE_AKTUELL",
  LUKKET = "LUKKET",
}
```

Reply:
Tja, `IKKE_AKTUELL` kan man vel kalle en vurdering, `UNNTAK` også, selv om vi har pleid å bare si registrere unntak /  sette unntak og ikke vurdere i denne sammenheng. Mens `KANDIDAT` er jo en systemgenerert hendelse, og LUKKET er noe vi har gjort manuelt (ikke noe veileder gjør). Så jeg synes typer av historikk passer fint :)